setup.py

```diff
@@ -154,13 +154,14 @@ def run(self) -> None:
 
         subprocess.check_call(["ls", "-hal"], cwd=source_dir, env=build_env)
         subprocess.check_call(["ls", "-hal", "build/lib"], cwd=source_dir, env=build_env)
-        subprocess.check_call(["ls", "-hal", "build/hw"], cwd=source_dir, env=build_env)
+        subprocess.check_call(["ls", "-hal", "runtime"], cwd=source_dir, env=build_env)
 
         tt_build_dir = self.build_lib + "/tt_lib/build"
         os.makedirs(tt_build_dir, exist_ok=True)
         self.copy_tree(source_dir / "build/lib", tt_build_dir + "/lib")
-        self.copy_tree(source_dir / "build/hw", tt_build_dir + "/hw")
+        self.copy_tree(source_dir / "runtime", tt_build_dir)
```

Comment:
I think this needs to be `runtime`? Otherwise it might copy everything inside recursively into `tt_build_dir`? Or am I reading this wrong?

Reply:
I think you're right.. either way, I don't think it'll work but I'll try that too. 

I'll turn this back to a draft since I'm unsure how to get this to work properly without setting up kernels after packaging like we are currently doing

Reply:
nvm, at the end of the day it was a path problem: https://github.com/tenstorrent/tt-metal/actions/runs/9322888631/job/25665558311